var/spack/repos/builtin/packages/py-torch/package.py

```diff
@@ -462,6 +462,9 @@ class PyTorch(PythonPackage, CudaPackage, ROCmPackage):
         working_dir="third_party/fbgemm",
     )
 
+    # Reduce parallelism during the build of flash_attn kernels (5GB+ per process)
```

Comment:
Can you add a comment link to the PR so we can update or remove the patch if/when it gets merged?